app/src/main/java/org/mozilla/fenix/components/appstate/AppStoreReducer.kt

```diff
@@ -39,6 +39,7 @@ internal object AppStoreReducer {
             topSites = action.topSites,
             recentBookmarks = action.recentBookmarks,
             recentTabs = action.recentTabs,
+            recentSyncedTabState = action.recentSyncedTabState,
```

Comment:
I'd been thinking that `AppAction.Change` was intended to expose a way to manipulate the entire `AppState` in a single action, for setting up initial states in tests or otherwise. If that wasn't the intended use-case, then I can remove `recentSyncedTabState` from the action entirely.

Reply:
We used to call `AppAction.Change` in `onStart` to populate the `HomeFragmentStore` with the cached state from various settings and storage. That was one of the reason it took so many params. We don't need to do this anymore since `AppStore` is global. Otherwise, it's current functionality is to change multiple parameters all at once easily in the BlocklistMiddle, but we will want to rethink what parameter needs to be exposed.

Reply:
Makes sense! I don't think it will need to be included for now, then